packages/components/src/components/input/component.tsx

```diff
@@ -56,15 +57,14 @@ export class KolInput implements Props {
 
 		return (
 			<Host
-				class={{
-					'kol-input': true,
+				class={clsx('kol-input', this.getModifierClassNameByMsgType(), {
```

Comment:
Finde `clsx` hier tatsächlich praktisch, verstehe aber auch, wenn wir es nicht als Dependency einführen wollen. Bisher wurde es nur in den Tests verwendet.

Habe _ohne_ `clsx` aber tatsächlich keine saubere Lösung parat, weil eine generated property nicht undefined sein darf. Folgendes ginge also nicht:
```
class={{
  [this.getModifierClassNameByMsgType()]: true,
}}
```